src/discord/discord.service.ts

```diff
@@ -1,75 +1,91 @@
-import { Injectable, HttpException, HttpStatus } from '@nestjs/common';
+import { HttpException, HttpStatus } from '@nestjs/common';
+import { createClient } from '@astrajs/collections';
+
 import { ReadDiscordDto } from './dto/read-discord.dto';
 import { UpdateDiscordDto } from './dto/update-discord.dto';
 
-@Injectable()
 export class DiscordService {
-  private discord: ReadDiscordDto[] = [];
-  create(createDiscordDto: ReadDiscordDto) {
+  astraClient;
+  usersCollection;
+  basePath = '/api/rest/v2/KEYSPACES/<namespace>/collections/<collectionName>';
+  constructor() {
+    (async () => {
+      try {
+        this.astraClient = await createClient({
```

Comment:
I think this should be moved to a provider and use NestJS config

- config https://docs.nestjs.com/techniques/configuration
- provider https://docs.nestjs.com/fundamentals/custom-providers#factory-providers-usefactory